packages/ovh-manager/navbar/navbar.service.js

```diff
@@ -0,0 +1,620 @@
+            title: this.$translate.instant('common_menu_means_mean'),
+            url: this.REDIRECT_URLS.paymentMeans,
+          },
+          (this.TARGET === 'EU' || this.TARGET === 'CA') && {
+            title: this.$translate.instant('common_menu_means_ovhaccount'),
+            url: this.REDIRECT_URLS.ovhAccount,
+          },
+          (this.TARGET === 'EU' || this.TARGET === 'CA') && {
+            title: this.$translate.instant('common_menu_means_vouchers'),
+            url: this.REDIRECT_URLS.billingVouchers,
+          }, {
+            title: this.$translate.instant('common_menu_means_refunds'),
+            url: this.REDIRECT_URLS.billingRefunds,
+          },
+          (this.TARGET === 'EU') && {
+            title: this.$translate.instant('common_menu_means_fidelity'),
+            url: this.REDIRECT_URLS.billingFidelity,
+          }, {
+            title: this.$translate.instant('common_menu_means_credits'),
+            url: this.REDIRECT_URLS.billingCredits,
+          }],
+        },
+
+        // Orders
+        (!currentUser.isEnterprise && this.TARGET === 'EU' && currentUser.ovhSubsidiary === 'FR') && {
+          title: this.$translate.instant('common_menu_orders'),
+          url: this.REDIRECT_URLS.orders,
+        },
+
+        // Contacts
+        (this.TARGET === 'EU') && {
+          title: this.$translate.instant('common_menu_contacts'),
+          url: this.REDIRECT_URLS.contacts,
+        },
+
+        // Tickets
+        {
+          title: this.$translate.instant('common_menu_list_ticket'),
+          url: this.REDIRECT_URLS.support,
+        },
+
+        // Logout
+        {
+          title: this.$translate.instant('global_logout'),
+          class: 'logout',
+          click: (callback) => {
+            this.ssoAuthentication.logout();
+
+            if (typeof callback === 'function') {
+              callback();
+            }
+          },
+        },
+      ],
+    };
+  }
+
+  getInternalLinks(currentUser, notificationsMenu) {
+    // Return login link if user not logged
+    if (!currentUser) {
+      return [{
+        title: this.$translate.instant('common_login'),
+        click: () => this.ssoAuthentication.logout(),
+      }];
+    }
+
+    const menu = [
+      this.getLanguageMenu(), // Language
+      this.getAssistanceMenu(currentUser.ovhSubsidiary), // Assistance
+      this.getUserMenu(currentUser), // User
+    ];
+
+    if (notificationsMenu.show) {
+      menu.splice(1, 0, notificationsMenu);
+    }
+    return menu;
+  }
+
+  getManagersNames() {
+    switch (this.TARGET) {
+      case 'EU': {
+        if (this.locale === 'FR') {
+          return ['portal', 'gamma', 'partners'];
+        }
+        return ['portal', 'gamma'];
+      }
+      case 'CA': {
+        return ['gamma'];
+      }
+      case 'US':
+      default: {
+        return [];
+      }
+    }
+  }
+
+  loadTranslations() {
+    return import(`./../translations/Messages_${this.$translate.use()}.xml`)
```

Comment:
Maybe I should split translations and create new translations files dedicated to the `navbar` in the `navbar` directory and use : 

```javascript 
return import(`./translations/Messages_${this.$translate.use()}.xml`)
```

Reply:
yes!